docs/attributes-registry/db.md

```diff
@@ -155,7 +155,7 @@ This group defines attributes for Azure Cosmos DB.
 | ------------------------------------ | ------ | --------------------------------- | -------------------------------------- | ---------------------------------------------------------------- |
 | `db.cosmosdb.client_id`              | string | Unique Cosmos client instance id. | `3ba4827d-4422-483f-b59f-85b74211c11d` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `db.cosmosdb.connection_mode`        | string | Cosmos client connection mode.    | `gateway`; `direct`                    | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `db.cosmosdb.operation_type`         | string | CosmosDB Operation Type.          | `Invalid`; `Create`; `Patch`           | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `db.cosmosdb.operation_type`         | string | CosmosDB Operation Type.          | `batch`; `create`; `delete`            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
nit: official branding has a space between "Cosmos DB"

I owe a branding review on this repo as there may be other places. Can handle this separately if it's easier

Reply:
Thanks! I believe I replaced all `CosmosDB` occurrences in the repo